composer.json

```diff
@@ -10,29 +10,29 @@
     "doctrine/orm": "~2.5.3",
     "ext-zip": "*",
     "geoip2/geoip2": "~2.4.2",
-    "doctrine/doctrine-bundle": "1.5.2",
+    "doctrine/doctrine-bundle": "~1.5.2",
     "mrclay/minify": "~2.3.0",
-    "jakeasmith/http_build_url": "^1",
+    "jakeasmith/http_build_url": "~1.0.0",
     "mobiledetect/mobiledetectlib": "~2.8.22",
-    "symfony/swiftmailer-bundle": "2.3.11",
+    "symfony/swiftmailer-bundle": "~2.3.11",
     "pear/archive_tar": "~1.4.2",
     "swiftmailer/swiftmailer": "~5.4.3",
-    "symfony/monolog-bundle": "2.8.2",
-    "sensio/distribution-bundle": "4.0.6",
-    "sensio/framework-extra-bundle": "3.0.10",
+    "symfony/monolog-bundle": "~2.8.2",
+    "sensio/distribution-bundle": "~4.0.6",
+    "sensio/framework-extra-bundle": "~3.0.10",
     "prestashop/translationtools-bundle" : "dev-master",
-    "tecnickcom/tcpdf": "6.2.12",
-    "composer/installers": "1.0.21",
-    "icanboogie/cldr": "1.3.9",
-    "incenteev/composer-parameter-handler": "~2.0",
-    "curl/curl": "1.2.1",
-    "ircmaxell/password-compat": "1.0.4",
+    "tecnickcom/tcpdf": "~6.2.12",
+    "composer/installers": "~1.0.21",
+    "icanboogie/cldr": "~1.3.9",
+    "incenteev/composer-parameter-handler": "~2.0.0",
+    "curl/curl": "~1.2.1",
+    "ircmaxell/password-compat": "~1.0.4",
     "ircmaxell/random-lib": "*",
-    "shudrum/array-finder": "1.1.0",
-    "phpoffice/phpexcel": "1.8",
-    "guzzlehttp/guzzle": "~4.0",
-    "csa/guzzle-bundle": "~1.3",
-    "paragonie/random_compat": "^1.4|^2.0",
+    "shudrum/array-finder": "~1.1.0",
+    "phpoffice/phpexcel": "~1.8.0",
+    "guzzlehttp/guzzle": "~4.0.0",
+    "csa/guzzle-bundle": "~1.3.0",
+    "paragonie/random_compat": "~1.4.0",
```

Comment:
This page explains that it is the same as double-pipe `||` or `OR`: http://qpleple.com/understand-composer-versions/

But why do that if we can't go for the `PHP 7.0` only version (`2.0`)?